Data/core_rulebook/cr__datacontrols.lua

```diff
@@ -0,0 +1,7127 @@
+-- Converted From LST file data\pathfinder\paizo\roleplaying_game\core_rulebook\cr__datacontrols.lst
```

Comment:
This whole file appears to be nonsense. :-) I wonder what it's useful for.

Reply:
its defining how the `FACT:` tags work. None of it is documented though.